app/controllers/spotlight/contact_forms_controller.rb

```diff
@@ -11,11 +11,7 @@ def new
 
     def create
       if @contact_form.valid?
-        if @contact_form.respond_to? :deliver_now
-          @contact_form.deliver_now
-        else
-          @contact_form.deliver
-        end
+        ContactMailer.report_problem(@contact_form).deliver_now
```

Comment:
We weren't doing it before (and things end up getting ugly; ActiveJob can't serialize `Spotlight::ContactForm`, so you end up tacking on a bunch of primitive arguments instead)